pom.xml

```diff
@@ -287,5 +287,25 @@
        <url>https://repo.jenkins-ci.org/public/</url>
      </pluginRepository>
   </pluginRepositories>
+
+  <profiles>
+    <profile>
+      <id>java8-doclint-disabled</id>
```

Comment:
I thought that the javadoc warnings were already 100% resolved.  Did I miss something?